Home page/timer

TheRealAndrxw opened the pull request:
Created Basic Home Page with simple timer, included slider for time adjustment and included break time once the timer has reached the half way mark.

kjlee2504 reviewed (changes requested):
Please in the description link to issues completed and update the “T” in the title to lower case. 
![IMG_6418](https://github.com/user-attachments/assets/89417cce-f38f-4a4d-a98e-b0494b029efa)

also there seems to be a bug where the 0 cannot be removed 
![Uploading IMG_6419.jpeg…]()

TheRealAndrxw commented:
I updated the timewise name to lowercase, and I fixed the break input so that when it is empty it recognizes as no break.
This pull request relates to issue #33 & issue #35 
It also relates to user story #34

kjlee2504 reviewed (approved):
LGTM!
